src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -185,9 +217,10 @@ private Card createJobCompletedCard() {
                 summary += " Back to Normal";
 
                 if (failingSinceRun != null) {
-                    long duration = run.getDuration() == 0L
-                            ? System.currentTimeMillis() - run.getStartTimeInMillis()
-                            : run.getDuration();
+
+                    if (run.getDuration() == 0L)
```

Comment:
Could you rewrite this, so facsBuilder.addBackToNormalTime uses factsBuilder new countCompletionTime 👍 

```java
factsBuilder.addBackToNormalTime(failingSinceRun.getStartTimeInMillis());
```

```java
public void addBackToNormalTime(long failedRunStartTime) {
       long currentBuildCompletionTime = countCompletionTime();
       long duration = currentBuildCompletionTime - failedRunStartTime;
       facts.add(new Fact(NAME_BACK_TO_NORMAL_TIME, TimeUtils.dateToString(duration)));
}
```

Reply:
Awesome suggestion, will change it now :) I also see other opportunities for improvement but for now I was just trying to fix notifications appearance.